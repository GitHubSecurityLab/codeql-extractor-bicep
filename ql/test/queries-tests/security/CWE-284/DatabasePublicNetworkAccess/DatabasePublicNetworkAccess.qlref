security/CWE-284/DatabasePublicNetworkAccess.ql
