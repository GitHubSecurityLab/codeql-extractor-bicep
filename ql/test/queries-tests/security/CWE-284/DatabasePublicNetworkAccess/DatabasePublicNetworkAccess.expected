| app.bicep:8:26:8:34 | String | Database 'vulnerable-sql-server' has public network access enabled, making it accessible from the internet. |
| app.bicep:19:26:19:34 | String | Database 'vulnerable-postgresql' has public network access enabled, making it accessible from the internet. |
| app.bicep:31:26:31:34 | String | Database 'vulnerable-mysql' has public network access enabled, making it accessible from the internet. |
